scope.cpp

```diff
@@ -1001,9 +1001,9 @@ void Scope::CheckCalibrationDuration(int currDuration)
 
     int rslt;
     CalstepDialog::GetCalibrationStepSize(pFrame->GetFocalLength(), pCamera->GetCameraPixelSize(),
-        pCamera->Binning, currSpd, CalstepDialog::DEFAULT_STEPS, 0.0, GetCalibrationDistance(), 0, &rslt);
+        pCamera->Binning, currSpdX, CalstepDialog::DEFAULT_STEPS, 0.0, GetCalibrationDistance(), 0, &rslt);
 
-    if (rslt == currDuration)
+    if (fabs(1.0 - (double)rslt/(double)currDuration) <=  0.20)

```

Comment:
since it is not obvious, could we add a comment here explaining what this means?